nextflow.config

```
@@ -52,6 +52,9 @@ params {
   cluster_weighting = "jaccard" // default weighting scheme for graph-based clustering
   nearest_neighbors = 20 // default nearest neighbors parameter for graph-based clustering
 
+  // Cell type annotation options
+  singler_label_name = "label.ont" // celldex reference label used for SingleR cell tyoe annotation
```

Originally just a typo fix, but if this is only really used in the reference workflow by my updated suggestions, maybe this?
```suggestion
  singler_label_name = "label.ont" // celldex reference label used for SingleR reference building
```